FastBank.Domain/RepositoryInterfaces/IUserRepository.cs

```diff
@@ -7,5 +7,11 @@ public interface IUserRepository
         public User? GetByEmail(string email);
 
         public void Add(User user);
+
+        public List<User> GetUserFriends(User user);
+
+        public void AddFriend(User user, User Friend);
```

Comment:
Why parameter Friend is Upper case here?

Reply:
Fixed